### PR TITLE
Fixes Access forbidden - State token missing

### PR DESCRIPTION
**Pull request by joselameira**
This commit error Access forbidden - State token missing while trying to login using Nextcloud Desktop.

Commit 99f7b730e7911459de22898e431cfcf525f0393a ignored the query part of the redirectUrl, needed for Nextcloud Desktop to succeed authentication.

### CODE REVIEW

**Review by julien-nc — APPROVED** [review]
:+1:  Thanks a lot for the fix.

**Comment by julien-nc** [comment]
Could you add a space between `fn` and `($k)` so the code style check passes? I'll merge right after.

**Comment by joselameira** [comment]
Hi @julien-nc

Space added!

**Pull request merged**
